docs/static/security/api-keys.asciidoc

```diff
@@ -2,8 +2,89 @@
 [[ls-api-keys]]
 ==== Grant access using API keys
 
-
 Instead of using usernames and passwords, you can use API keys to grant
 access to {es} resources. You can set API keys to expire at a certain time,
 and you can explicitly invalidate them. Any user with the `manage_api_key`
 or `manage_own_api_key` cluster privilege can create API keys.
+
+{ls} can send both collected data and monitoring information to {es}. If you are
+sending both to the same cluster, you can use the same API key. For different
+clusters, you need to use an API key per cluster.
```

Comment:
I think we should start with "API keys are tied to the cluster they were created in", and then go on to explain that in a single cluster a key can be shared for ingestion+monitoring purposes, while a cluster production cluster + monitoring cluster setup will require separate keys.

Reply:
I can add "API keys are tied to the cluster they were created in" now. 

Seems like the "single cluster a key can be shared for ingestion+monitoring purposes, while a cluster production cluster + monitoring cluster setup will require separate keys." piece becomes relevant at 7.9 when we support API keys for monitoring and management. WDYT?

Reply:
I think we can just mention it as you suggested for now and I am OK in deferring more details to a next release.